Examples/test-suite/typescript_optional.i

```diff
@@ -0,0 +1,61 @@
+%module typescript_optional
+
+
+%inline %{
+template <class T>
+class OptionalValue
+{
+  public:
+    T v;
+};
+%}
+
+%typemap (typescriptoptional, optional="1") OptionalValue<nm::D> "nm::D";
+//%typemap (typescriptoptional, optional="1") OptionalValue<nm::A> "nm::A";
```

Comment:
Best not to leave any commented out code, especially without explaining why is it commented out.